packages/api/package.json

```diff
@@ -12,11 +12,20 @@
     "@aws-sdk/client-s3": "^3.36.0",
     "@aws-sdk/lib-dynamodb": "^3.36.0",
     "@aws-sdk/client-sqs": "^3.36.0",
+    "@middy/core": "^2.5.1",
+    "@middy/http-event-normalizer": "^2.5.1",
+    "@middy/http-json-body-parser": "^2.5.1",
+    "@middy/validator": "^2.5.1",
     "@types/aws-lambda": "^8.10.83",
+    "ajv": "^8.6.3",
+    "ajv-formats": "^2.1.1",
+    "http-errors": "^1.8.0",
```

Comment:
Are you still using `http-errors` somewhere? If not might want to remove `http-errors` and `@types/http-errors`.

Reply:
Good catch